src/Uno.UWP/System.Runtime/WindowsRuntimeBufferExtensions.cs

```diff
@@ -8,85 +8,73 @@ namespace System.Runtime.InteropServices.WindowsRuntime
 	public static class WindowsRuntimeBufferExtensions
 	{
 		public static IBuffer AsBuffer(this byte[] source)
+			=> AsBuffer(source, 0, source.Length, source.Length);
+		
+		public static IBuffer AsBuffer(this byte[] source, int offset, int length)
+			=> AsBuffer(source, offset, length, length);
+
+		public static IBuffer AsBuffer(this byte[] source, int offset, int length, int capacity)
 		{
 			if (source is null)
 			{
 				throw new ArgumentNullException(nameof(source));
 			}
 
-			return source.AsBuffer(0, source.Length);
-		}
+			if (offset < 0) throw new ArgumentOutOfRangeException(nameof(offset), "offset is less than 0 (zero).");
+			if (length < 0) throw new ArgumentOutOfRangeException(nameof(length), "length is less than 0 (zero).");
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6307767)